lib/chef-dk/skeletons/code_generator/templates/default/kitchen.yml.erb

```diff
@@ -5,11 +5,9 @@ driver:
 provisioner:
   name: chef_zero
 
-# Uncomment the following verifier to leverage Inspec instead of Busser (the
-# default verifier)
-# verifier:
-#   name: inspec
-#   format: doc
+verifier:
+  name: inspec
+  format: doc
```

Comment:
Oh cool! Let's see how it looks in a Kitchen run once we have a nightly then. That's awesome.